### ISSUE TITLE
Issue with simfphys

### ISSUE THREAD

**Issue by bronemishka** [NanoAi/gm_apg#11]
After unghost all symfphys cars become purple.

**Comment by AvarianKnight** [comment]
Can you give me your sh_config.lua and the car you're using that this happens with?

**Comment by NanoAi** [comment]
This is actually a bug having to do with setting the colour/transparency of
the simfphys cars. Not sure why the bug happens but this colour bug has
been around forever basically.

**Comment by NanoAi** [comment]
Can anyone with Symfphys cars try running the following code and see if it fixes the cars turning purple?
```
local ENT = FindMetaTable( "Entity" )
ENT._SetColor = ENT._SetColor or ENT.SetColor

function ENT:SetColor( color, ... )
	local color = color
	local r, g, b, a

	if type(color) == "number" then
		color = Color(color, select(1, ...) or 255, select(2, ...) or 255, select(3, ...) or 255)
	else
		if type(color) == "table" and not IsColor(color) then
			r = color.r or color[1] or 255
			g = color.g or color[2] or 255
			b = color.b or color[3] or 255
			a = color.a or color[4] or 255
			color = Color(r, g, b, a)
		end
	end

	assert( IsColor(color), "Invalid colour passed to SetColor!\nThis error\
	prevents stuff from turning purple/pink." )

	return self:_SetColor( color )
end
```

**Comment by AvarianKnight** [comment]
This should be fixed with #13 @NanoAi

**Comment by NanoAi** [comment]
Fixed in #13
